frontend/components/organization/Edit.jsx

```diff
@@ -108,6 +109,23 @@ function Edit({ actions }) {
 
   const getSandboxMsg = days => <span>{sandboxMsg(days)}</span>;
 
+  function getInvitationSentMsg() {
```

Comment:
Lets move this function out of the component and then pass in `mostRecentlyAddedUser` as an arg.

Reply:
I had it like that originally. I moved it here for parallelism with the similarly-used `getSandboxMsg` right above it.

Reply:
`getSandboxMsg` is still taking an argument (rather than relying on variables outside the function scope)

Reply:
This took an argument too until the linter flagged it as an error because `mostRecentlyAddedUser` was already defined. I could give the argument a different name to fix it but moving it where Andrew suggested is probably clearer.